databricks/koalas/testing/utils.py

```diff
@@ -53,6 +55,8 @@ def conf(cls):
 
     @classmethod
     def setUpClass(cls):
+        if SparkContext._active_spark_context is not None:
+            SparkContext._active_spark_context.stop()
```

Comment:
Seems multiple contexts can be initiated when running doctests first.
I think It does not happen in PySpark testing because it runs them in each process.